django/db/models/base.py

```diff
@@ -1275,9 +1275,9 @@ def check(cls, **kwargs):
             errors += [
                 *cls._check_index_together(),
                 *cls._check_unique_together(),
-                *cls._check_indexes(),
                 *cls._check_ordering(),
-                *cls._check_constraints(databases),
+                *cls._check_indexes(databases),
```

Comment:
Is there any reason to move `_check_indexes()`?

Reply:
There is no particular reason for this change. At previous commits I try to address errors that showed up in GitHub check but not in my computer. I'll revert that.